dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -390,6 +409,64 @@ class BGPPeerMgr(Manager):
         return peers
 
 
+def prefix_attr(attr, value):
+    if not value:
+        return None
+    else:
+        try:
+            prefix = netaddr.IPNetwork(str(value))
+        except:
+            return None
+    return str(getattr(prefix, attr))
+
+
+class InterfaceMgr(Manager):
+    def __init__(self, daemon, directory, interface_table = swsscommon.CFG_INTF_TABLE_NAME):
```

Comment:
We would also want to consider VLAN_INTERFACE as well - CFG_VLAN_INTF_TABLE_NAME

Reply:
Thanks for your suggestion. Have added VlanInterfaceMgr.